Specs/NumberWithUnit/Korean/DimensionModel.json

```diff
@@ -0,0 +1,838 @@
+[
```

Comment:
Are there any Korea-specific units that are common enough the recognizers should support?

Reply:
The Korean specific units will be added with the implementation.